src/components/resources/Resources.js

```diff
@@ -10,28 +10,28 @@ export default function Home() {
         Adopting pets is a much more humane way to find your familys newest best
         friend. Every year around 6.5 million companion pets enter shelters in
         just the USA, and thats only dogs and cats. Out of those
-        <b>6.5 million</b>, only <b>3.2 million</b> are adopted and
-        <b>1.5 million</b> are euthanized (Statistics provided by
+        <span> 6.5 million</span>, only <span>3.2 million</span> are adopted and
+        <span> 1.5 million</span> are euthanized (Statistics provided by
         <a
           target="_blank"
           rel="noreferrer"
           href="https://www.aspca.org/animal-homelessness/shelter-intake-and-surrender/pet-statistics"
         >
-          ASPCA
+          {""} ASPCA
         </a>
         ).
         <br />
```

Comment:
Think this whole block could just in one `<p>` . Be sure to remove the `<br/>` tag. You could probably add a margin bottom